src/ansys_sphinx_theme/__init__.py

```diff
@@ -463,6 +378,137 @@ def configure_theme_logo(app: Sphinx):
         theme_options["logo"] = logo_option
 
 
+def convert_pdf_to_png(pdf_path: pathlib.Path, output_dir: pathlib.Path, output_png: str):
+    """
+    Convert PDF to PNG images.
+
+    Parameters
+    ----------
+    pdf_path : pathlib.Path
+        Path to the PDF file.
+    output_dir : pathlib.Path
+        Path to the output directory.
+    output_png : str
+        Name of the output PNG file.
+    """
+    try:
+        doc = pymupdf.open(pdf_path)  # open document
+        for page in doc:
+            pix = page.get_pixmap()
+            pix.save(os.path.join(output_dir, output_png))
+    except RuntimeError as e:
+        raise RuntimeError(f"Failed to convert PDF to PNG: {e}")
+
+
+def add_cheat_sheet(
+    app: Sphinx, pagename: str, templatename: str, context: Dict[str, Any], doctree: document
+) -> None:
+    """Add a cheat sheet to the left navigation sidebar.
+
+    Parameters
+    ----------
+    app : ~sphinx.application.Sphinx
+        Application instance for rendering the documentation.
+    pagename : str
+        Name of the current page.
+    templatename : str
+        Name of the template being used.
+    context : dict
+        Context dictionary for the page.
+    doctree : ~docutils.nodes.document
+        The doctree.
+    """
+    cheatsheet_options = app.config.html_theme_options.get("cheatsheet", {})
+    pages = cheatsheet_options.get("pages", ["index"])
+    pages = [pages] if isinstance(pages, str) else pages
+    if cheatsheet_options and any(pagename == page for page in pages):
+        sidebar = context.get("sidebars", [])
+        sidebar.append("cheatsheet_sidebar.html")
+        context["sidebars"] = sidebar
+
+
+def build_quarto_cheatsheet(app: Sphinx):
+    """
+    Build the Quarto cheatsheet.
+
+    Parameters
+    ----------
+    app : sphinx.application.Sphinx
+        Application instance for rendering the documentation.
+    """
+    cheatsheet_options = app.config.html_theme_options.get("cheatsheet", {})
+
+    if not cheatsheet_options:
+        return
+
+    cheatsheet_file = cheatsheet_options.get("file", "")
+    output_dir = cheatsheet_options.get("output_dir", "_static")
+
+    if not cheatsheet_file:
+        return
+
+    cheatsheet_file = pathlib.Path(app.srcdir) / cheatsheet_file
+    file_name = str(cheatsheet_file.name)
+    file_path = cheatsheet_file.parent
+    output_dir_path = pathlib.Path(app.outdir) / output_dir
+    try:
+        # Add the cheatsheet to the Quarto project
+        result = subprocess.run(
+            ["quarto", "add", "ansys/pyansys-quarto-cheatsheet@v1", "--no-prompt"],
+            cwd=file_path,
+            capture_output=True,
+            text=True,
+        )
```

Comment:
Let's isolate the version within the scope of this function:

```suggestion
        VERSION = "v1"
        result = subprocess.run(
            ["quarto", "add", f"ansys/pyansys-quarto-cheatsheet@{VERSION}", "--no-prompt"],
            cwd=file_path,
            capture_output=True,
            text=True,
        )
```